CMakeLists.txt

```diff
@@ -666,18 +652,17 @@ if(NOT CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
 
   # Note: For Valgrind support.
   genex_for_option(DEBUG)
-  target_compile_options(devilution PUBLIC $<${DEBUG_GENEX}:-fno-omit-frame-pointer>)
+  target_compile_options(${BIN_TARGET} PUBLIC $<${DEBUG_GENEX}:-fno-omit-frame-pointer>)
 
   # Warnings for devilutionX
   target_compile_options(${BIN_TARGET} PRIVATE -Wall -Wextra -Wno-unused-parameter)
 
   # For ARM and other default unsigned char platforms
-  target_compile_options(devilution PRIVATE -fsigned-char)
   target_compile_options(${BIN_TARGET} PRIVATE -fsigned-char)
 endif()
 
 if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
-  target_compile_options(devilution PRIVATE "/W0")
+  target_compile_options(${BIN_TARGET} PRIVATE "/W0")

```

Comment:
_Really_  hard to review this file...

Reply:
This is OK, now there is just one target

Reply:
I wasn't specifically talking about this particular change, but the file as a whole. Should've added a file-level comment, sorry for the confusion.